dbaas/maintenance/registered_functions/functools.py

```diff
@@ -103,45 +105,27 @@ def get_infra_password(host_id):
 
 
 def get_host_user(host_id):
-    """Return HOST_USER"""
+    """Return HOST_PASSWORD"""
     from physical.models import Host
-    host = Host.objects.filter(id=host_id).select_related('cs_host_attributes')
 
     try:
-        host = host[0]
-    except IndexError as e:
+        host = Host.objects.get(id=host_id)
+    except ObjectDoesNotExist as e:
```

Comment:
How about change for except Host.DoesNotExist

Doing that you dont need to import ObjectDoesNotExist